pkg/spanconfig/spanconfigkvsubscriber/testdata/system_span_configs

```diff
@@ -0,0 +1,127 @@
+
+start
+----
+
+update
+upsert [a,c):A
+upsert [d,f):D
+----
+
+updates
+----
+[/Min,/Max)
+[a,c)
+[d,f)
+
+store-reader key=a
+----
+conf=A
+
+store-reader key=e
+----
+conf=D
+
+store-reader key=g
+----
+conf=MISSING
+
+update
+upsert {entire-keyspace}:+X
+----
+
+updates
+----
+[/Min,/Max)
+
+store-reader key=a
+----
+conf=A+X
+
+store-reader key=e
+----
+conf=D+X
+
+# The system span config we set should still apply.
+store-reader key=g
+----
+conf=MISSING+X
+
+# Update span configs that target tenant keyspaces and ensure handlers are
```

Comment:
Move everything below this line into a separate file; the spans don't overlap with the special [alphabet, alphabet) style spans listed above, except for `{source=1, target=1}:` which could stay in this file.